libs/base/ui/overlays/drawer/src/component.ts

```diff
@@ -4,7 +4,7 @@ import { TAG_NAME } from './tag';
 export const drawerComponent = componentDef({
   name: TAG_NAME,
   impl: () =>
-    window.HTMLDialogElement
```

Comment:
let's make it explicit by adding `isServer` check to the mix (but keep current behavior)